DeepSpeech/fr/import_cvfr.sh

```diff
@@ -25,6 +25,10 @@ pushd $HOME/ds/
 
 		tar -C /mnt/extracted/data/cv-fr/ -xf /mnt/sources/cv-3-fr.tar.gz
 
+		create-corpora -d /mnt/extracted/corpora -f /mnt/extracted/data/cv-fr/validated.tsv -l fr -s ${DUPLICATE_SENTENCE_COUNT}
+
+		mv /mnt/extracted/corpora/fr/*.tsv /mnt/extracted/data/cv-fr/
```

Comment:
this `mv` should also be part of what is done only if ` ${DUPLICATE_SENTENCE_COUNT} > 1`